velox/dwio/parquet/tests/reader/ParquetTableScanTest.cpp

```diff
@@ -1148,6 +1148,19 @@ TEST_F(ParquetTableScanTest, readParquetColumnsByIndex) {
   EXPECT_THROW(
       AssertQueryBuilder(op).split(split).copyResults(pool()),
       VeloxRuntimeError);
+
+  // Schema Evolution
+  rowType = ROW({"c1"}, {{BIGINT()}});
```

Comment:
Would it be worth to also test if the schema has one more column than the parquet file (and scan spec will use the default value?)

Reply:
I am going to add a bunch of tests like you suggesed tomorrow. I wanted to ensure CI is happy with the fix first.

Reply:
I added the test you suggested. Other combinations are already present.